Project Management Process.md

```diff
@@ -0,0 +1,287 @@
+# IPFS Project Management Process
+
+**Status: work in progress, not even a full draft yet**
+
+## Introduction
+TODO: describe the purpose of this document
+
+- this is an opt-in process, nobody in the community is *required* to use this process
```

Comment:
However, it also should not be exclusive. It is important to preserve a view of things that members of the community can quickly jump into and hack to contribute.